gallery/visualtests/nxDrawer.js

```diff
@@ -16,8 +16,8 @@ describe('NxDrawer', function() {
 
     await waitAndGetElements(`#${drawerId}`);
 
-    // Wait for animation.
-    await wait(200);
+    // Wait for animation and focus.
+    await wait(300);
```

Comment:
I'd rather have a wider margin.  If it failed sometimes at 200, I feel like it might still occasionally fail at 300. Let's bump it up to 500.

Reply:
> I'd rather have a wider margin. If it failed sometimes at 200, I feel like it might still occasionally fail at 300. Let's bump it up to 500.

Extended it to 500 here: b9fd128d795458fa31b68ac18d0208c3275ef686